recipes/setup_shclustering.rb

```diff
@@ -111,7 +111,7 @@
 
 ruby_block 'captain elected' do
   block do
-    node.normal['splunk']['shclustering']['captain_elected'] = true
+    node.default['splunk']['shclustering']['captain_elected'] = true
```

Comment:
this `ruby_block` sets the `node.normal` precedence on purpose. A splunk search head captain bootstrap procedure should only occur one time and one time only. Therefore, setting `node.normal` here is appropriate, since that value needs to be persisted on all future chef runs once the execute resource has bootstrapped the captain. If this is reverted to `node.default`, future runs that call helper method `#ok_to_bootstrap_captain?` will always be false.

Reply:
cookstyle barks about using normal.
```
recipes/setup_shclustering.rb:114:5: R: Chef/Correctness/NodeNormal: Do not use node.normal. Replace with default/override/force_default/force_override attribute levels. (https://docs.chef.io/workstation/cookstyle/chef_correctness_nodenormal)
    node.normal['splunk']['shclustering']['captain_elected'] = true
```

Reply:
that is why there has been an exception in `.rubocop_todo.yml`:

Reply:
I can't think of another way of doing this so I just used an in line ignore for cookstyle.

Reply:
what is wrong with using what was in the repo before -- `.rubocop.yml` and `.rubocop_todo.yml`?

Reply:
IMHO, inline ignores are code-smells. Hunting them down in the future is a maintenance nightmare.

Reply:
My thought is that then your ignoring the whole file so other places where it may not be absolutely needed or incorrectly used would also be ignored instead of the 1 specific edge case where it is needed.

Reply:
reverted my changes and moved my changes to a new fork https://github.com/TheREK3R/chef-splunk/pull/1